src/fiatconnect/saga.test.ts

```diff
@@ -1796,6 +1835,44 @@ describe('Fiatconnect saga', () => {
         .returns('0x12345')
         .run()
     })
+    it.each(Object.keys(CryptoType) as CryptoType[])(
```

Comment:
I verified this test fails for CELO with the old implementation. It also happens to fail for cREAL

Reply:
nit: combine this test with the test above? I am guessing the above tests the same thing with cUSD

Reply:
yeah I largely copied the above test.

Reply:
I'll delete the other one and update the name of this one to make it more clear what it's testing